README.rst

```diff
@@ -92,7 +92,7 @@ Usage
 
     # get boolean result if motion detection
     # is enabled or not
-    cam.is_motion_detection_enabled  # True
+    base.is_motion_detection_enabled  # True
```

Comment:
@broox @tchellomello There were arlo cameras which doesn't require base station. Those cameras also have motion detection capabilities (enable/disable motion detection as well).  So, for those sort of cameras, leaving this property is_motion_detection_enabled in the cam object itself would make more sense right ?  Now since it has been moved to base, not sure 100% if it would still work for those sort of cameras.  I don't have that camera, but got this problem when i made some change before w.r.t HASS

Reply:
is_motion_detection_enabled should be in base, not cameras.  At least for Arlo and Arlo Pros.  The is_motion_detection_enabled is synonymous with setting the entire system to Armed or Disarmed.  I can attest to this.  I have different boolean switches set to each camera for this property.  If I turn off one, it deactivates detection on all cameras.  If I turn off all of them and then turn on one, it activates detection on all of them.  It's a base station setting.  It confused me for quite some time why I kept getting alerts.  It's good to have an Arm/Disarm functionality in the library, but for individual control you'll need to turn each camera off / enable privacy.

Have there been any discussions on combining efforts with https://github.com/jeffreydwalter/arlo ?

Reply:
@viswa-swami `is_motion_detection_enabled` was never defined on the `ArloCamera` class, so this example in the README did not work. It is, however, defined on `ArloBaseStation.is_motion_detection_enabled`, so I simply updated the README to use the working, defined property.

If `ArloBaseStation.is_motion_detection_enabled` is something we want, I can submit it as a new issue/feature request?